docs/implementation_plans/2026-03-15_category_default_profiles_agents.md

```
@@ -0,0 +1,348 @@
+    TaskData --> AiConfigInferenceProfile : profileId
+    AgentIdentityEntity --> AgentStateEntity : currentStateId
+    AgentStateEntity --> Task : activeTaskId
+    AgentIdentityEntity --> AgentTemplateEntity : via TemplateAssignmentLink
+```
+
+---
+
+## 4. Implementation Steps (Ordered)
+
+### Step 1: Data Model Changes ✅
+| File | Change | Status |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)